guides/source/getting_started.md

```diff
@@ -194,6 +165,38 @@ of the files and folders that Rails creates by default:
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more information about ignoring files.|
 |.rubocop.yml|This file contains the configuration for RuboCop.|
 |.ruby-version|This file contains the default Ruby version.|
+|.devcontainer/|This folder contains the Dev Container configuration|
+
+### Opening the Blog Application in a Dev Container
+
+A Dev Container is a fully functional development environment running in a container
+on your machine. Our new Rails application comes with a Dev Container already
+configured and ready to use.
+
+We will use VSCode to spin up and work with our Dev Container. First we need to open
+our app in VSCode. This can be done by opening VSCode and then opening our application's
+folder, or directly from the command line within our application's directory with:
+
+```bash
+$ code .
+```
```

Comment:
I'm not sure that we need to explain _how_ to open a file in VSCode as it feels a bit too verbose, maybe we can just say to launch the VSCode application? I don't feel strongly about this if you feel that its better to go into the detail.

Reply:
I think this is a good tip for new folks who don't yet know command line prompts. I say we keep it.

Reply:
I actually just realized the `code` command is not added to your path automatically when you install vscode. So I am going to remove this, other wise I will need to explain how to create it - which involves opening vscode 😅 so it's kind of a side quest at that point.

Reply:
But I tried to simplify it.